widget/embedded/package.json

```diff
@@ -37,6 +37,7 @@
     "rango-types": "^0.1.57",
     "react-i18next": "^12.2.0",
     "react-router-dom": "^6.8.0",
+    "react-virtuoso": "^4.6.2",
```

Comment:
As I noticed we have lots of libs for handling virtualized lists including: `react-virtuoso`, `react-window`, `react-window-infinite-loader`, `react-virtualized-auto-sizer`. If the new lib is better (bundle size, performance, easy to use, etc) can we make our virtualized list component to use it?